package.json

```diff
@@ -25,32 +25,47 @@
     "Web Risk API"
   ],
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "^1.9.0"
   },
   "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^12.0.0",
+    "codecov": "^3.0.0",
```

Comment:
I'm surprised we need to install codecov here

Reply:
dependency removed, good call, thanks!